src/pipx/main.py

```diff
@@ -588,7 +592,10 @@ def _add_run(subparsers: argparse._SubParsersAction) -> None:
     p.add_argument(
         "--python",
         default=DEFAULT_PYTHON,
-        help="The Python version to run package's CLI app with. Must be v3.6+.",
+        help=(
+            "Python to run with. Possible values can be the executable name (python3.11), "
+            "the version to pass to py launcher (3.11), or the full path to the executable."
```

Comment:
See above comment

Reply:
I think better `v3.8+`, since version _3.7_ will (hopefully) be dropped soon.

Reply:
Thinking of maintenance, how about adding a constant like `MINIMUM_PYTHON_VERSION` to [constants.py](https://github.com/pypa/pipx/blob/main/src/pipx/constants.py), which could then be imported into `main.py`?

Reply:
Good point!

Reply:
Thanks for the suggestion @dukecat0 and @chrysle! I added a `MINIMUM_PYTHON_VERSION` constant.